src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,97 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    static final int DEFAULT_CAPACITY = 16;
+    static final float DEFAULT_LOAD_FACTOR = 0.75F;
+    private int size;
+    private int threshold;
+    private int capacity;
+    private Node<K,V>[] table;
+
+    public MyHashMap() {
+        table = new Node[DEFAULT_CAPACITY];
+        capacity = DEFAULT_CAPACITY;
+        threshold = (int) (capacity * DEFAULT_LOAD_FACTOR);
+    }
 
     @Override
     public void put(K key, V value) {
-
+        if (size >= threshold) {
+            table = resize();
+        }
+        int index = getIndex(key);
+        Node<K, V> current = new Node<>(key, value, null);
+        Node<K, V> nodeInTable = table[index];
+        if (table[index] == null) {
+            table[index] = current;
+            size++;
+        } else {
+            while (nodeInTable.next != null || Objects.equals(nodeInTable.key, key)) {
+                if (Objects.equals(nodeInTable.key, key)) {
+                    nodeInTable.value = value;
+                    return;
```

Comment:
```suggestion
                    break;
```
Let's replace that with break.

Reply:
Воно не буде працювати))))) Так як там є ще додаткова логіка, тому тут краще викорисати return, а не ускладнювати if-else конструкцію

Reply:
+